src/js/components/TabPanesComponent.jsx

```diff
@@ -94,6 +94,29 @@ var TabPanesComponent = React.createClass({
     return tabs[0].id;
   },
 
+  getClearLinkForFilter: function (filterQueryParamKey) {
+    var state = this.state;
+
+    if (state[filterQueryParamKey].length === 0) {
+      return null;
+    }
+
+    let router = this.context.router;
+    let currentPathname = router.getCurrentPathname();
+    let query = Object.assign({}, router.getCurrentQuery());
+    let params = Object.assign({}, router.getCurrentParams());
```

Comment:
Hmm, don't get it, why this ```params``` is Object.assigned. It isn't modified somewhere, so I think there's no need for it.